### PR TITLE
pdf2image 1.17.0 

### PR DESCRIPTION
**Pull request by msentissi**
pdf2image 1.17.0 

**Destination channel:** Defaults

### Links

- [PKG-6297]
- dev_url:        https://github.com/Belval/pdf2image/tree/v1.17.0

### Explanation of changes:

- upgrade to v1.17.0

[PKG-6297]: https://anaconda.atlassian.net/browse/PKG-6297?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: recipe/meta.yaml [hunk 1]
```diff
@@ -1,37 +1,48 @@
...
   home: https://github.com/Belval/pdf2image
+  doc_url: https://belval.github.io/pdf2image
+  dev_url: https://github.com/Belval/pdf2image
   license: MIT
   license_family: MIT
-  license_file: LICENSE.txt
-  summary: A wrapper around the pdftoppm and pdftocairo command line tools to convert PDF to a PIL Image list.
+  license_file: LICENSE
+  summary: A python (3.7+) module that wraps pdftoppm and pdftocairo to convert PDF to a PIL Image object
+  description: A python (3.7+) module that wraps pdftoppm and pdftocairo to convert PDF to a PIL Image object
```
Comment (cbouss) [inline recipe/meta.yaml]:
```suggestion
  summary: A python module that wraps pdftoppm and pdftocairo to convert PDF to a PIL Image object
  description: A python module that wraps pdftoppm and pdftocairo to convert PDF to a PIL Image object
```

**Review by cbouss — APPROVED** [review]
LGTM. I verified that our current poppler does have the pdftoppm utility.

File: recipe/meta.yaml [hunk 2]
```diff
@@ -1,37 +1,48 @@
...
+    - python
     - pip
-    - python >=3
+    - setuptools
+    - wheel
   run:
-    - python >=3
+    - python
     - pillow
+    - poppler
```
Comment (ifitchet) [inline recipe/meta.yaml:33]: Should we comment this to say that `poppler` is a requirement as documented in the `README.md` (and not in `setup.py`)?

Unless it is obvious to everyone else!  😆

**Review by ifitchet — APPROVED** [review]

File: recipe/meta.yaml (hunk 1 above)
Comment (msentissi) [reply recipe/meta.yaml]: Thank you for checking!

File: recipe/meta.yaml (hunk 2 above)
Comment (msentissi) [reply recipe/meta.yaml:33]: It's not! When dependencies are only present in less "standard" places, making that explicit seems like a good idea, done.

File: recipe/meta.yaml [hunk 3]
```diff
@@ -1,37 +1,50 @@
...
 
 test:
   imports:
     - pdf2image
+  requires:
+    - pip
+  commands:
+    - pip check
+    - pdftoppm -v
```
Comment (ifitchet) [inline recipe/meta.yaml]: `pdftoppm` is a command in `poppler`, why is this package testing it?  In fact this package looks it up in `poppler`: https://github.com/Belval/pdf2image/blob/master/tests.py#L68

We would want to text a command that this package creates, wouldn't we?  (Assuming there is one..., maybe `pdfinfo`? https://github.com/Belval/pdf2image/blob/master/tests.py#L38)

**Comment by msentissi** [comment]
> LGTM. I verified that our current poppler does have the pdftoppm utility.

@ifitchet I added the pdftoppm check to automate the verification that @cbouss did manually.

File: recipe/meta.yaml (hunk 3 above)
Comment (ifitchet) [reply recipe/meta.yaml]: Actually, `pdfinfo` is part of `poppler` see https://conda-metadata-app.streamlit.app/Search_by_file_path (dynamic pages don't let me capture having typed `pdfinfo` into the dialog box)

**Comment by ifitchet** [comment]
> @ifitchet I added the pdftoppm check to automate the verification that @cbouss did manually.

Although, as noted previously, https://github.com/Belval/pdf2image/blob/master/tests.py#L68, `tests.py` effectively does that test by searching for it on the PATH.

**Comment by msentissi** [comment]
@cbouss @ifitchet I switched to GH as a source to include tests and patched out 7 (+4 on windows) failing tests. One of them was failing because it wasn't being skipped on `osx` when it should have been for example. I decided to skip it along with the others without attempting to fix each one individually. 
The remaining (~100 passing) tests still indicate that `poppler` is installed properly and contains the required components.

**Pull request merged**